data/card_catalogue.json

```diff
@@ -748,177 +790,97 @@
         {
           "title": "Reload",
           "upgrade": "Tactical Reload",
-          "tier": "grey",
+          "tier": "blue",
           "details": {
-            "rules": "Return a Card with Blue Rarity from your discard pile to your hand",
+            "rules": "Return a Card with Blue Rarity to your hand",
             "flavour": "Don't retreat, reload.",
             "image": "assets/core/card/illustration/Reload.png"
           }
         },
         {
           "title": "Tactical Reload",
           "upgrade": "Cowboy Reload",
-          "tier": "blue",
+          "tier": "green",
           "details": {
-            "rules": "Return a Card with Green Rarity from your discard pile to your hand",
-            "flavour": "A tactical reload is the action of reloading a weapon that has only fired a few rounds out of its magazine, and retaining the original magazine",
+            "rules": "Return a Card with Green Rarity to your hand",
+            "flavour": "A tactical reload is the action of reloading a weapon that has only figold a few rounds out of its magazine, and retaining the original magazine",
             "image": "assets/core/card/illustration/Tactical Reload.png"
           }
         },
         {
           "title": "Cowboy Reload",
           "upgrade": "Expert Reload",
-          "tier": "green",
+          "tier": "red",
           "details": {
-            "rules": "Return a Card with Red Rarity from your discard pile to your hand",
-            "flavour": "Cowboy's only load up 5 of the 6 bullet chambers. In the 6th chamber, they leave a hundred dollar bill to pay the undertaker.",
+            "rules": "Return a Card with Red Rarity to your hand",
+            "flavour": "Cowboy's only load up 5 of the 6 bullet chambers. In the 6th chamber, they leave a hundgold dollar bill to pay the undertaker.",
             "image": "assets/core/card/illustration/Cowboy Reload.png"
           }
         },
         {
           "title": "Expert Reload",
           "upgrade": "",
-          "tier": "red",
+          "tier": "gold",
           "details": {
-            "rules": "Return a Card with Gold Rarity from your discard pile to your hand",
+            "rules": "Return a Card with Gold Rarity to your hand",
             "flavour": "“Peace is the time it takes to reload your rifle.” - Bob Dylan",
             "image": "assets/core/card/illustration/Expert Reload.png"
           }
         },
         {
           "title": "Adlib",
           "upgrade": "Improvise",
-          "tier": "grey",
+          "tier": "blue",
           "details": {
-            "rules": "Add Three Grey Rarity Cards to your hand. Return them to the deck at the end of the turn.",
+            "rules": "Draw 3 grey cards",
```

Comment:
Grey cards are all reactive, so forcing you to instant cast blows. Getting 3 free grey cards per session isn't a big deal, they can just keep them if they want.